- tag_id: 'GPS_only'
  latitude: 40.2460444
  longitude: -111.6471631
- tag_id: 'GPS_only'
  latitude: 40.2461618
  longitude: -111.6470753
- tag_id: '1'
  latitude: 40.2462257
  longitude: -111.6469283
- tag_id: '2'
  latitude: 40.2461336
  longitude: -111.6469292
- tag_id: '3'
  latitude: 40.2460437
  longitude: -111.6469265
- tag_id: 'mallet'
  latitude: 40.2460492
  longitude: -111.6470403
- tag_id: 'bottle'
  latitude: 40.2461199
  longitude: -111.6470251